.github/workflows/ci.yml

```diff
@@ -53,3 +53,21 @@ jobs:
       - run: pnpm run build
       - run: pnpm run test:coverage
       - uses: codecov/codecov-action@v3.1.4
+  test-types:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        node: ['18','20']
```

Comment:
Not sure if Node.js versions makes any difference. Perhaps? In a way TypeScript should work the same way on all supported Node.js versions.

Did you consider `tstyche --target 5.0,current` to test types using TypeScript 5.0 and currently version? This is useful in if you want to check that TypeScript 5.0 (or any other lowest supported version) really works.

Reply:
It's was our default configuration in case there is any issue with the next node.js lts version. Thanks for taking the time to review and sorry to come back so late.

And maybe we will try to add a matrix of TypeScript version later if it worth it.